ERP_INES/Data/Modules/Finance/Repositories/Concrete/PSQLPaymentMethodRepository.cs

```diff
@@ -0,0 +1,80 @@
+using ERP_INES.Data.Modules.Finance.Repositories.Interfaces;
+using ERP_INES.Domain.Modules.Finance.Entities;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.EntityFrameworkCore;
+
+namespace ERP_INES.Data.Modules.Finance.Repositories.Concrete;
+
+public class PSQLPaymentMethodRepository:IPaymentMethodRepository
+{
+    
+    private readonly FinanceDbContext _context;
+
+    public PSQLPaymentMethodRepository(FinanceDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<List<PaymentMethod>> GetPaymentMethodsAsync()
```

Comment:
Is possible that method will never been accessed, please check if is need to be removed